### PR TITLE
Can use staff of storms on station

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
If you're not on lavaland, you can instead summon localized ash storms by room/area

The staff can now be edited to summon any kind of storm, just in case we ever have a distant future in which more of them are sprited/coded

:cl: Kor
add: The Staff of Storms, dropped by Legion, can now be used on station.
/:cl:

### CODE REVIEW

**Comment by Kromgar** [comment]
I suggested this to Kor an hour ago. He works damn fast

**Comment by francinum** [comment]
good shit

**Comment by KorPhaeron** [comment]
It turns out I have a poor grasp of the new weather subsystem/it might not be written for what I have in mind, so stay tuned for whether this can exist or not

**Comment by Kromgar** [comment]
My dreams nooo

**Comment by KorPhaeron** [comment]
alright it works

**Comment by KorPhaeron** [comment]
Current issues: If you use this in the same area multiple times, it spams invisible storms

**Comment by KorPhaeron** [comment]
@Xhuis

**Comment by WJohn** [comment]
I was hoping this thing could call meteors down, that'd be pretty cool.

**Comment by GunHog** [comment]
Or new storm types. Plasma storm that spawns random tesla zaps and EMPs on random mobs; Comes with a lightning blue version red/brass colored lighting effects that happen during Nar'Sie vs Ratvar. Or improving rad storms to have effects.

**Comment by jughu** [comment]
Only if you let the ash storm die on lavaland when the legion is defeated and add some special snowflake text about the storm that dispeared

**Comment by KorPhaeron** [comment]
Okay it works now

File: code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm [hunk 1]
```diff
@@ -141,40 +141,44 @@ Difficulty: Medium
...
+		A.target_z = user.z
+		A.telegraph_duration = 100
+		A.end_duration = 100
 		A.telegraph()
-
-	storm_cooldown = world.time + 600
+	user.visible_message("<span class='warning'>[user] holds [src] skywards as red lightning crackles into the sky!</span>", \
+	"<span class='notice'>You hold [src] skyward, calling down a terrible storm!</span>")
+	playsound(user, 'sound/magic/Staff_Change.ogg', 200, 0)
+	A.telegraph()
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm]: Won't this double telegraph with line 177?

**Comment by optimumtact** [comment]
ARE YOU SURE IT WORKS NOW

**Pull request merged**